File: test-project/greeter.provider/src/test/java/examples/greeter/ScriptingTest.java
package examples.greeter;

import javax.script.*;
import org.junit.jupiter.api.*;
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.*;

class ScriptingTest {

    @Test
    void testScripting() {
        ScriptEngineManager manager = new ScriptEngineManager();
        assertThat(manager.getEngineFactories(), not(nullValue()));
    }
}

File: test-project-kotlin/greeter.provider/src/test/kotlin/examples/greeter/ScriptingTest.kt
package examples.greeter

import javax.script.*
import org.junit.jupiter.api.*
import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.*

class ScriptingTest {
    @Test
    fun testScripting() {
        val manager = ScriptEngineManager()
        assertThat(manager.getEngineFactories(), not(nullValue()))
    }
}
